examples/people-to-csv.py

```diff
@@ -0,0 +1,25 @@
+"""Example to scrape a list of Companies, and put overviews in csv form"""
+
+from scrape_linkedin import CompanyScraper
+import pandas as pd
+
+# LIST YOUR COMPANIES HERE
+my_company_list = [
+    'facebook', 'mit-sloan-school-of-management', 'linkedin',
+    'harvard-university'
+]
+
+company_data = []
+
+with CompanyScraper() as scraper:
+    # Get each company's overview, add to company_data list
+    for name in my_company_list:
+        sc = scraper.scrape(company=name, people=True)
+        overview = sc.overview
+        overview['company_name'] = name
```

Comment:
optional: The overview already has a `name` field that seems redundant with this. The ID that is being saved here is more typically referred to as an "id" or "slug", which may be more appropriate field names if you need to save this.